requirements.txt

```diff
@@ -10,3 +10,4 @@ tldextract>=2.0.1
 feedfinder2>=0.0.4
 jieba3k>=0.35.1
 python-dateutil>=2.5.3
+tinysegmenter==0.3
```

Comment:
Do you mean `>=0.3` ? Let's merge anyways but add a todo to change this if needed